contrib_bots/lib/github_issues.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+import requests
+import json
+from future import standard_library
+standard_library.install_aliases()
+import urllib.request, urllib.error, urllib.parse
+import six.moves.configparser
+import os
```

Comment:
Conventionally there is an expected grouping of Python imports: standard library imports, related third party imports and local application/library specific imports:

https://www.python.org/dev/peps/pep-0008/#imports

Also having them in alphabetical order will look good too.